### ISSUE TITLE
Properties dialog missing window floating hints [sway wm]

### ISSUE THREAD

**Issue by Mel34** [linuxmint/nemo#2643]
```
 * Nemo version (nemo --version) 4.8.4
 * Is issue with desktop or windowed nemo? windowed nemo
 * Distribution - Arch
 * Graphics hardware *and* driver used - Rx580 open source drivers
 * 32 or 64 bit - 64 bit
 ```

**Issue**
Properties dialogue windows doesn't provide hints that allows Sway to treat them as floating windows.

**Steps to reproduce**
Run Nemo under Sway window manager
Open Properties on some file/folder

**Expected behaviour**
A floating Properties window is displayed (works on Thunar, Nautilus)
Instead Properties window created by Nemo is tiled.

**Other information**

**Comment by mtwebster** [comment]
Are you referring to the file properties window? (right-click->properties on a file)

Doing a quick search, it may be this issue:
https://github.com/swaywm/sway/issues/4655

Are you using wayland?

It appears as though I must disable resizing for this window. However, the window in thunar can be resized and you said it's fine.

We're calling this a GtkDialog, honestly I would think that's enough.

This doesn't seem like something nemo should have to worry about though if I could get more information I'd be happy to do something to make it work here.

**Comment by Mel34** [comment]
Yes I am, it is the file properties window just as you described. And yes, sway is a wayland compositor.

**Comment by Aleksanaa** [comment]
Still not fixed yet, hyprland 0.25.0
